pr_agent/servers/github_app.py

```diff
@@ -93,7 +93,7 @@ async def handle_request(body: Dict[str, Any]):
         api_url = pull_request.get("url")
         if not api_url:
             return {}
```

Comment:
**Suggestion:** Consider adding a log message when a pull request URL is not found in the webhook event. This could help with debugging and understanding the system's behavior.
```suggestion
        if not api_url:
            logging.warning('Pull request URL not found in webhook event.')
            return {}
```